### ISSUE TITLE
Timeouts.

### ISSUE THREAD

**Issue by pjvalla** [EttusResearch/uhd#288]
## Issue Description ##
Using these branches:

gnuradio : maint-3.7
gr-ettus : master
uhd : UHD-3.14
uhd-fpga : UHD-3.14

Running simple flowgraph on X310 using 10 Gige ports (XG build loaded on the FPGA)

## Setup Details ##
Simple flow graph shown below:
![Workspace 1_040](https://user-images.githubusercontent.com/10789547/60349178-18629e00-998f-11e9-9ff8-386e070acb62.png)

## Expected Behavior ##
The system should just produce 2 output plots occasional overflows are acceptable at high sample rates.

## Actual Behaviour ##
Works for a while (< 2 minutes ) then stops outputting data and the system produces timeouts.  Restarting the flowgraph results in this error message:

RuntimeError: EnvironmentError: IOError: Block ctrl (CE_03_Port_60) no response packet - AssertionError: bool(buff)
  in uint64_t ctrl_iface_impl<_endianness>::wait_for_ack(bool, double) [with uhd::endianness_t _endianness = (uhd::endianness_t)0u; uint64_t = long unsigned int]

The FPGA must be reloaded to get the system back to a usable state.

## Update 

Loaded stock ```usrp_x310_fpga_XG.bit``` and ran this flow graph:

![Workspace 1_041](https://user-images.githubusercontent.com/10789547/60351120-100c6200-9993-11e9-9862-c6feb6a7e04f.png)

Same exact issue.  System times out and needs a fresh load of the FPGA to get a working system.

## Steps to reproduce the problem ##
Should be able to recreate flowgraph **no** custom RFNoC modules being used. 

## Additional Information ##
After looking at internal noc_shell signals, the dataflow seems to be shut-off.

**Comment by natetemple** [comment]
@pjvalla  

Can you try updating to the latest stable release and try it on your machine? 3.14.1.0

Can you also try setting the `rmem_default` and `wmem_default` buffers in addition to the `rmem_max` and `wmem_max` that UHD will suggest:

```
sudo sysctl -w net.core.rmem_max=6250000
sudo sysctl -w net.core.wmem_max=6250000
sudo sysctl -w net.core.rmem_default=6250000
sudo sysctl -w net.core.wmem_default=6250000
```

One other test to try, in the `device3` block, try hard setting the master clock try by adding the device arg: `master_clock_rate=200e6`

**Comment by michael-west** [comment]
This issue is presumed to be resolved on UHD 4.0.0.0.  Feel free to re-open if it is not.
